maplibre-adapter/src/main/java/com/amplifyframework/geo/maplibre/view/MapLibreView.kt

```diff
@@ -154,16 +171,119 @@ class MapLibreView
                         BitmapFactory.decodeResource(resources, defaultPlaceActiveIcon)
                     )
                 }
-                this.symbolManager = SymbolManager(this, map, it, null, null).apply {
-                    iconAllowOverlap = true
-                    iconIgnorePlacement = true
+
+                // Clear the current symbols from the map since a new SymbolManager will be created
+                if (this::symbolManager.isInitialized) {
+                    this.symbolManager.deleteAll()
+                }
+
+                removeClusterLayers(it)
```

Comment:
If the clusterlayers are already set should be remove them and recreate or just leave them alone?

Reply:
If clustering is enabled with different clustering options, then we'll need to update the properties for the cluster circle and number layers, but adding a layer to the style that has the same ID as an existing layer on the style throws an error, so I removed the old cluster circle and number layers (which also needs to be done when disabling clustering).